tests/2015-08-01/Microsoft.Compute.tests.json

```diff
@@ -90,22 +86,82 @@
         }
       }
     },
-
+    
+    {
+      "name": "extensions - Unsupported",
```

Comment:
I suppose we should still allow the resource definitions which follow the old extension schema, right?